contrib/babelfishpg_common/sql/coerce.sql

```diff
@@ -143,3 +143,15 @@ CREATE FUNCTION sys.varchar_to_name(pg_catalog.varchar)
 RETURNS pg_catalog.name
 AS 'babelfishpg_common', 'pltsql_text_name'
 LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+-- int8 -> money
+CREATE FUNCTION sys.int8_to_money(INT8)
```

Comment:
what will this INT8 resolve to? pg_catalog or sys?

Reply:
pg_catalog